agriculture/static/js/dashboard.js

```diff
@@ -63,7 +63,8 @@ const INFO_WINDOW_CONTENT_CONTROLLER = "" +
     "            <span class='digi-icon-color fas fa-wind fa-2x'></span>" +
```

Comment:
Don´t know how to put this comment in source3.zip and source4.zip, so I put it here: Those files should not be committed.